cluster/addons/fluentd-elasticsearch/fluentd-es-image/Makefile

```diff
@@ -4,7 +4,7 @@ IMAGE = fluentd-elasticsearch
 TAG = 1.11
```

Comment:
The minor version number is purely arbitrary -- increment to your heart's content.